src/status_im/contexts/keycard/unblock/events.cljs

```diff
@@ -24,12 +25,15 @@
          [:dispatch [:open-modal :screen/keycard.unblock.ready-to-unblock]]]}))
 
 (rf/reg-event-fx :keycard/unblock.phrase-entered
- (fn [{:keys [db]} [{:keys [seed-phrase]}]]
-   {:db (assoc-in db [:keycard :unblock :masked-phrase] seed-phrase)
-    :fx [[:dispatch [:navigate-back]]
-         [:dispatch
-          [:open-modal :screen/keycard.pin.create
-           {:on-complete #(rf/dispatch [:keycard/unblock.pin-created (security/mask-data %)])}]]]}))
+ (fn [{:keys [db]} [{:keys [seed-phrase key-uid]}]]
+   (if (not= key-uid (get-in db [:keycard :application-info :key-uid]))
+     {:fx [[:dispatch
+            [:enter-seed-phrase/set-error (i18n/label :t/seed-phrase-dont-match-migrate-flow)]]]}
```

Comment:
isn't that a part of your another PR? i don't see :enter-seed-phrase/set-error in the develop

Reply:
hi @flexsurfer, yes PR merged only an hour ago - https://github.com/status-im/status-mobile/pull/22252